### PR TITLE
Support postgresql 14

### PR DESCRIPTION
**Pull request by Leo-XM-Zeng**

### CODE REVIEW

**Comment by JelteF** [comment]
I think the easiest is to change those specific tests to a python test and skip them for PG14. i.e. create a new file `test/pycheck/temporary_table_test.py`.

To be clear we won't have time to review this in detail or merge this before the 0.1.0 release. But given the small number of necessary changes, I think this can definitely be targeted for 0.2.0.

Thank you for your contribution

**Comment by Leo-XM-Zeng** [comment]
> I think the easiest is to change those specific tests to a python test and skip them for PG14. i.e. create a new file `test/pycheck/temporary_table_test.py`.
> 
> To be clear we won't have time to review this in detail or merge this before the 0.1.0 release. But given the small number of necessary changes, I think this can definitely be targeted for 0.2.0.
> 
> Thank you for your contribution

Understood. Thank you for your guidance

File: test/pycheck/temporary_table_test.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from .utils import Cursor, PG_MAJOR_VERSION, eprint
+
+
+def test_temporary_table(cur: Cursor):
```
Comment (JelteF) [inline test/pycheck/temporary_table_test.py:4]:
```suggestion
def test_temporary_table_set_access_method(cur: Cursor):
```

Comment (Leo-XM-Zeng) [reply test/pycheck/temporary_table_test.py]: Agreed, modified

**Comment by JelteF** [comment]
Did you close this on purpose? I was planning to merge this soon-ish.

**Comment by Leo-XM-Zeng** [comment]
> Did you close this on purpose? I was planning to merge this soon-ish.

No, no, I see a lot of changes, I plan to reorganize and resubmit.

**Comment by JelteF** [comment]
Ah alright. Sounds good, yeah we're refactoring a bunch of things to make the C vs C++ division clearer.

**Comment by Leo-XM-Zeng** [comment]
> Ah alright. Sounds good, yeah we're refactoring a bunch of things to make the C vs C++ division clearer.

This is good, I also want to try to do a little better, let you misunderstanding, sorry.

**Pull request closed without merging**